src/org/usfirst/frc/team467/robot/Autonomous/Actions.java

```diff
@@ -164,20 +141,14 @@ public static boolean moveDistanceComplete(double distance) {
 		}
 	}
 
-	public static ActionGroup doNothing(){
-		ActionGroup mode = new ActionGroup("none");
-		mode.addAction(nothing());
-		return mode;
-	}
-
```

Comment:
We used to use this as the final autonomous step - doNothing forever. Is this no longer needed?

Reply:
There is a different nothingForever() action.